### ISSUE TITLE
would be great to customize private links

### ISSUE THREAD

**Issue by dkm** [cozy/cozy-photos-v2#93]
Currently, private links are not intended to be understandable by "humans". It would be great to be able to set a private name that could be an alias to the hash-based one (like vimeo does)
For example, when shared, an album could be reached by:

toto.cozy..../public/albums/SHA1 or toto.cozy/public/albums/my-name

That would be easier to remember :)

**Comment by frankrousseau** [comment]
Thank you for the idea. We do not work on the FIle application for the moment. But that something we'll include in our thinking about sharings.

**Comment by dkm** [comment]
I was thinking about the photo app. Is it still the case that you don't work on it atm?

**Comment by frankrousseau** [comment]
Yes, we are focusing on our email client and the Home module currently.

**Comment by dkm** [comment]
Ok... too bad for the photo app :( Just so you know, all ppl that I try to have a look at cozy do not even try because there is no android/ios sync app... And it's true that uploading a file from a phone is quite cumbersome... But that OT and I guess you already know that :) Thanks for the good work !
